release-lock-file-spec.md

```diff
@@ -23,52 +152,69 @@ single JSON object.
 
 ## Filename
 
-When creating a release, convention is to name this document `<version>.json`.
+When creating a release, convention is to name this document using one of the following conventions.
 
-* The `1.0.0` Release would be named `1.0.0.json`
+* In cases where multiple *Release Lockfiles* are being store alongside one another, convention is to use `<version>.json`.  For example, the `1.0.0` release would be named `1.0.0.json`.  
+* In cases where a single *Release Lockfile* is being stored, convention is to use `lock.json`.
 
-Package managers may keep their own copy of this file as part of their package installation process.  In these cases it is up to the package manager to decide on a naming convention.  Package managers **should** keep the `.json` file extension.
+Package managers **should not** rely on these naming conventions nor are they
+required to follow them.
```

Comment:
So all of these statements use the **should** keywords which are meant to say, if you don't use this pattern, we really recommend that you have a pretty good reason.  

Including these types of things helps with general consistency across implementation in areas like this where we don't **need** compatibility but that it's nice from the user perspective for things to at least be familiar when they switch between tools.